lib/tlCore/OSWin32.cpp

```diff
@@ -62,22 +62,33 @@ namespace tl
             Windows getWindowsVersion()
             {
                 Windows out = Windows::Unknown;
+#if _WIN32_WINNT >= 0x0A00
                 if (IsWindows10OrGreater())
                 {
                     out = Windows::_10;
                 }
-                else if (IsWindows8Point1OrGreater())
+                else
+#endif
+#if _WIN32_WINNT >= 0x0603
+		  if (IsWindows8Point1OrGreater())
```

Comment:
It looks like something happened to this formatting, the "if" is indented extra spaces?

Reply:
It is the standard formatting in emacs to indicate it continues from the else above.

Reply:
Ah good point, I was not thinking about the "else" above. If the indentation is changed to four spaces I am OK with it.

The interleaving of "#if" and "if" statements is a bit hard to read, but this code is mostly static so it's not a big deal.